lib/ldap_fluff/posix_member_service.rb

```diff
@@ -52,4 +55,12 @@ class UIDNotFoundException < LdapFluff::Error
 
   class GIDNotFoundException < LdapFluff::Error
   end
+
+  private
+
+  def user_group_filter(uid, user_dn)
+    unique_filter = Net::LDAP::Filter.eq('uniquemember', user_dn) &
+                    Net::LDAP::Filter.eq('objectClass', 'groupOfUniqueNames')
```

Comment:
There's also `groupOfNames`. Should that also be supported? We search for it here:
https://github.com/theforeman/ldap_fluff/blob/77a387a50d64565bf1ca4c0e26f8fb6ed9a58614/lib/ldap_fluff/posix.rb#L24

Reply:
There are multiple group-like object types in LDAP. I think it's a high level decision what we support (and document!)